#!/usr/bin/env python
# field of view configuration

PACKAGE="shape_finder"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("lean_tolerance", double_t, 1, "lean_tolerance", 5.0, 0.0, 90.0)

exit(gen.generate(PACKAGE, "shape_finder", "GravityNodelet"))
